larky/src/main/java/com/verygood/security/larky/modules/vgs/cerebro/text/analyzer/analyzers/NoopTextPIIAnalyzer.java

```diff
@@ -0,0 +1,27 @@
+package com.verygood.security.larky.modules.vgs.cerebro.text.analyzer.analyzers;
+
+import com.verygood.security.larky.modules.vgs.cerebro.text.analyzer.dto.TextPIIEntity;
+import com.verygood.security.larky.modules.vgs.cerebro.text.analyzer.spi.TextPIIAnalyzer;
+import net.starlark.java.eval.EvalException;
+import net.starlark.java.eval.Starlark;
+import net.starlark.java.eval.StarlarkFloat;
+
+import java.util.List;
+
+public class NoopTextPIIAnalyzer implements TextPIIAnalyzer {
+  @Override
+  public List<TextPIIEntity> analyze(String text, String language, List<String> entities, StarlarkFloat scoreThreshold)
+      throws EvalException {
+    throw Starlark.errorf("cerebro.pii_analyzer.analyze method must be overridden");
```

Comment:
Wouldn't noop means no operation? It wouldn't throw exceptions. It will just return empty results right?

Reply:
Renamed to NotImplementedTextPIIAnalyzer